adb_graphics/default_specs.yml

```diff
@@ -480,13 +471,38 @@ dewp: # Dew point temperature
     wind: 10m
 finedust:
   int:
-    <<: *coldust
+    clevs: [1, 4, 7, 11, 15, 20, 25, 30, 40, 50, 75, 150, 250, 500]
```

Comment:
You could add a PyYAML anchor here and reference it later in full_int_dust to reduce duplication, like was done for coldust before this change.

Reply:
Thank you Christina!